maintainers/maintainer-list.nix

```diff
@@ -8097,6 +8097,12 @@
     githubId = 592876;
     name = "Robert W. Pearce";
   };
+  rpgwaiter = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add rpgwaiter``.